integrations/extensions/starter-kits/zendesk-guide/README.md

```diff
@@ -0,0 +1,141 @@
+# Zendesk Guide article search
+
+## Background
+
+This is a starter kit for accessing the Zendesk "Search Articles" API, which is part of the Zendesk Guide feature.  It is one of two starter kits we have for working with Zendesk.  The other is the [Zendesk Support](https://github.com/watson-developer-cloud/assistant-toolkit/tree/master/integrations/extensions/starter-kits/zendesk-support) starter kit, which enables submitting and querying support tickets for the Zendesk Support feature.
+
+The OpenAPI spec in this starter kit includes the following endpoint:
+
+- `GET /api/v2/help_center/articles/search`: Search for articles relevant to a given query.
+
+That endpoint is described in detail at https://developer.zendesk.com/api-reference/help_center/help-center-api/search/#search-articles.
+
+## Basic vs Advanced
+The `basic` starter kit implements a single yet useful API call for retrieving articles. If you are new to custom extensions, we recommend you start there to get an idea on how to use custom extensions with skills. The `advanced` folder contains a kit with the more advanced functionality of API parameters for filtering your search results, which you can use as a creative springboard for more complex use cases.
+
+## Pre-Requisite Steps
+
+Follow the steps listed in [Pre-Req: Getting Auth Keys and Configuring Your Server](https://github.com/watson-developer-cloud/assistant-toolkit/tree/master/integrations/extensions/starter-kits/zendesk-support#pre-req-getting-auth-keys-and-configuring-your-server) to configure Zendesk to let you access its APIs.
+
+## Other Setup Info
+
+### Setup in a new Assistant
+
+If you want to make a _new_ Assistant using this starter kit, take the following steps:
+
+- Download the OpenAPI specification (`zendesk-article-search-openapi.json`) and Actions JSON file (`zendesk-article-search-actions.json`) from the `basic` folder in this starter kit.
+   - This section assumes the basic version of the starter kit, which will provide a basic Zendesk article search capability. For Zendesk filtered search, use the advanced version of the starter kit by downloading the files from the `advanced` folder.
```

Comment:
I think the setup for basic and advanced is kind of confusing here, because in this sub-bullet you tell people they can use the advanced subdirectory instead, but then in the later bullets, you are still referring to the basic filenames.  Maybe it is better to just have the first bullet list both options and then have the later steps that refer to file names list both options as well.  Or maybe it is better to just leave the advanced stuff out of this section and have a different section altogether for advanced.  Anyway, I would recommend thinking about how this reads for both basic and advanced a little more.

Reply:
I agree. I moved it up to the top level as a parenthetical in case that is less confusing. I didn't make a separate advanced section for the setup because the steps are the same.